lib/index.js

```diff
@@ -17,7 +17,7 @@ module.exports = (options = {}) => {
 
     tree.match({tag: 'include'}, node => {
       let src = node.attrs.src || false;
-      let locals = node.attrs.locals || false;
+      let locals = node.attrs.locals || (node.content ? node.content.join().replace(/'\n'/g, '') : false);
```

Comment:
I think it should be replaced according to the changes in the comment to

```js
let locals = node.attrs.locals ?? node.content?.join().replace(/\n/g, '');
```
> ⚠️ Pay attention to the regular expression, there are optional single quotes

Reply:
nice, i fixed the regex. wouldn't we need an ESx compiler (babel?) to user this syntax? the "npm run test" gets broken with error:

```js
✖  25:47  Parsing error: Unexpected token ? 
```

Reply:
> wouldn't we need an ESx compiler (babel?) to use this syntax?

Yes, which version of the node are you using? This package starts support from 10 and up

Reply:
v10.16.0

Reply:
yeahh, unfortunately they are supported from 14 version